extension.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "SemanticMediaWiki",
+	"version": "3.0.0-alpha",
+	"author": [
+        "[https://www.mediawiki.org/wiki/User:Jeroen_De_Dauw Jeroen De Dauw]",
+        "James Hong Kong",
+        "[http://korrekt.org Markus Krötzsch]",
```

Comment:
Feel free to change the order, I just wanted to get this out of my git box so others can play with it.